account-management-app/src/main/webapp/WEB-INF/jspx/includes/storage-provider-form.jspx

```diff
@@ -61,8 +61,23 @@
                  cssClass="error"
                  element="div" />               
               </li>
-	            
-	            <c:if test="${param.storageProviderType eq 'AMAZON_S3' || param.storageProviderType eq 'AMAZON_GLACIER'}">
+
+							<c:if test="${param.storageProviderRole eq 'secondary'}">
+								<li>
+									<form:label
+										cssErrorClass="error"
+										path="${param.storageProvider}.properties['WRITABLE_BY_NON_ROOT']">
+										<spring:message code="storageprovider.writablebynonrootuser"/>
+									</form:label>
+									<form:checkbox
+										cssErrorClass="error"
+										path="${param.storageProvider}.properties['WRITABLE_BY_NON_ROOT']"
+										value="true"
```

Comment:
If this param name is changed on the DuraCloud side, it'll need to be changed here (and above) as well

Reply:
👍